packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2411,34 +2403,62 @@ App User Profile schema properties have the following standard [JSON Schema Draf
 | type                           | type of property                           | `string`, `boolean`, `number`, `integer`, `array` | FALSE    | FALSE  | FALSE    |
 | enum                           | enumerated value of the property           | array                                             | TRUE     | TRUE   | FALSE    |
 | oneOf                          | non-empty array of valid JSON schemas      | array                                             | TRUE     | TRUE   | FALSE    |
+| format                         | validates the format type of the property      | string                                            | TRUE     | TRUE   | FALSE    |
 
 ##### Description Details
 
-* `enum`: The value of the property is limited to one of the values specified in the `enum` definition.  The list of values for `enum` has to be made up of unique elements.
-
-* `oneOf`: Okta only supports `oneOf` for specifying display names for an `enum`. Each schema has the following format:
-
- ```json
-{
-  "const": "enumValue",
-  "title": "display name"
-}
- ```
+- `enum`: The value of the property is limited to one of the values specified in the `enum` definition. The list of values for the `enum` must be made up of unique elements.
 
-In case `enum` is used in conjunction with `oneOf`, the set of enumerated values and their order must be kept.
+- `oneOf`: Okta only supports `oneOf` for specifying display names for an `enum`. Each schema has the following format:
 
-```json
-{"enum": ["S","M","L","XL"],
-  "oneOf": [
-    {"const": "S", "title": "Small"},
-    {"const": "M", "title": "Medium"},
-    {"const": "L", "title": "Large"},
-    {"const": "XL", "title": "Extra Large"}
-  ]
-}
- ```
-
-`oneOf` is only supported in conjunction with `enum`, providing a mechanism to return a display name for the `enum` value.
+  ```json
+  {
+    "const": "enumValue",
+    "title": "display name"
+  }
+  ```
+
+  In case `enum` is used in conjunction with `oneOf`, the set of enumerated values and their order must be kept.
+
+  ```json
+  {"enum": ["S","M","L","XL"],
+    "oneOf": [
+      {"const": "S", "title": "Small"},
+      {"const": "M", "title": "Medium"},
+      {"const": "L", "title": "Large"},
+      {"const": "XL", "title": "Extra Large"}
+    ]
+  }
+  ```
+
+  `oneOf` is only supported in conjunction with `enum`, providing a mechanism to return a display name for the `enum` value.
+
+- `format`: Okta supports the `format` keyword and validates the following format attributes: `uri`, `date-time`, `email`, `ref-id`, `encrypted`, `hashed`, `country-code`, `language-code`, `locale`, and `timezone`. See the following request that uses the format property:
```

Comment:
Hi @benzhao It was included in the JIRA ticket from support.

Reply:
I don't think we should add documentation that we support this because I don't think there's validation for most, if not all of this in custom schemas. I recall there was a bug for country-code not being validated in custom schemas, but it looks like that was reverted for whatever reason.